packages/expect/src/types.ts

```diff
@@ -44,6 +44,7 @@ export type MatcherState = {
   isExpectingAssertions?: boolean;
   isNot: boolean;
   promise: string;
+  snapshotState: any;
```

Comment:
There's `SnapshotStateType` in `jest-snapshot`

Reply:
Good catch!